src/facewidgetscene.cpp

```diff
@@ -1,4 +1,5 @@
 /*
```

Comment:
**Reporter**: sneha-nd
**Rule**: build/include_subdir
**Severity**: WARN
**File**: src/facewidgetscene.cpp L21

Include the directory when naming .h files

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<1660455139>*